stages/org.osbuild.wsl.conf

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python3
+"""
+Configure advanced settings in Windows Subsystem for Linux.
+
+The stage configures the WSL settings on the system.
+"""
+
+import sys
+
+import iniparse
+
+import osbuild.api
+
+SCHEMA = """
+"additionalProperties": false,
+"description": "WSL configuration.",
```

Comment:
Please add `"additionalProperties": false,`. Otherwise one could make a typo in the property and the schema would still validate. I would also suggest to specify `"minProperties"` to `1` in each object, since executing the stage without any options doesn't do anything useful.